### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by meyerchak** [shiningw/ncdownloader#42] [General]
Thank you for making NCdownloader.  I really like Nextcloud and your addon is a great addition. 

Ubuntu 20.04, Intel nuc i3: youtube-dl install instructions caused errors. I needed to use “sudo pip install youtube-dl” here is a great article from LinuxBabe. [install YouTube-dl](https://www.linuxbabe.com/desktop-linux/install-youtube-dl-linux-use)

The speed of the download to NC is my only issue. YouTube-dl 60 kbps.

IDEA:
I am wondering if you can make an option to use yt-dlp.

thank you!

Cheers

**Comment by shiningw** [comment]
Hi,
 yt-dlp is included in the app. if you remove all versons of youtube-dl from your system, it will use yt-dlp

**Comment by meyerchak** [comment]
wow. very cool.

thank you!
